WeatherToday/Sources/Data/NetworkAPI/Models/APIWeather.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  APIWeather.swift
+//  WeatherToday
+//
+//  Created by Taher on 7/10/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+struct WeatherApi: Weather, Decodable {
+
+    let temperature: Double
+    let humidity: Double
+
+    enum CodingKeys: String, CodingKey {
```

Comment:
```suggestion
    enum CodingKeys: String, CodingKey {
    
```

Reply:
resolved in this [commit](https://github.com/nimblehq/ci-cd-sample-ios/pull/20/commits/d19faa8fbaec66b5f882dd4d38f85f4a4ca738c1)